internal/testplanet/planet.go

```diff
@@ -158,11 +159,17 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 			}))
 
 		go func(n *Node) {
+			if err := n.Discovery.Bootstrap(ctx); err != nil {
+				log.Error(err.Error())
+			}
 			// refresh the interval every 500ms
 			t := time.NewTicker(500 * time.Millisecond).C
```

Comment:
why 500ms?

Reply:
I'm not sure, I actually didn't set that.